### DISCUSSION TITLE
nerdctl 1.3.1 kubernetes version compatibility?

### DISCUSSION THREAD

**Discussion by mcbrad** [containerd/nerdctl#2202] [Q&A] [answered]
We are slowing working our clusters up to to K8 1.24 but are currently on K8 1.19.16. We are working to convert to containerd with our net upgrade to K8 1.21.14. Checking what version of kubernetes is required for the latest nerdctl.

Thanks!

**Comment by AkihiroSuda** [comment]
> Checking what version of kubernetes is required for the latest nerdctl.

What do you mean?
Kubernetes is not required for nerdctl.

**Comment by mcbrad** [comment]
Sorry I was referring to the containerd version included. I saw that version 1.7.0 only works with certain K8 versions.

**Reply by AkihiroSuda** [reply, marked as answer]
https://github.com/containerd/nerdctl/releases/tag/v1.3.1

> This release of nerdctl is expected to be used with containerd v1.6 or v1.7.
